english/Find_the_cow.md

```diff
@@ -0,0 +1,3 @@
+Can you [find the cow ?] (https://github.com/udacity/create-your-own-adventure/blob/master/english/Find_the_cow.md)
+If not [go back](https://github.com/udacity/create-your-own-adventure/blob/master/english/marshmallow.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.